report/main.Rmd

```diff
@@ -0,0 +1,313 @@
+---
+title: "Effect of Interventions on Tampering Sugary Beverages"
+date: "`r Sys.Date()`"
+output: 
+   pdf_document:
+      latex_engine: xelatex
+      keep_tex: true
+header-includes:
+  - \usepackage{fancyhdr}
+  - \pagestyle{fancy}
+  - \fancyhead[L]{STAT 550}
+  - \fancyhead[R]{Consultation Report}
+  - \fancyfoot[C]{\thepage}
+  - \renewcommand{\headrulewidth}{2pt}
+  - \renewcommand{\footrulewidth}{1pt}
+  - \setmainfont{Times New Roman} 
+  - \usepackage{xcolor}
+  - \usepackage{setspace}\singlespacing
+---
+
+
+```{r setup, include = FALSE}
+library(tidyverse)
+library(lubridate)
+library(rlang)
+library(visdat)
+library(corrplot)
+library(gridExtra)
+library(grid)
+library(viridis) 
+library(forcats)
+source("../code/EDA.R")
+source("../code/ITSA.R")
+source("../code/GEE.R")
+source("../code/LMEM.R")
+```
+
+**Contributions:**\linebreak
+
+
+```{=latex}
+\textbf{Parham Pishrobat (71097927):} Introduction, Data, ITSA, Other EDA Plots, Formatting\\ \\ \\
+\textbf{Sarah Masri (97415681):} \quad \quad GEE, Missing Values, Conclusion, Formatting\\ \\ \\
+\textbf{Johnson Chen (85784080):}  \quad LMEM, Correlation, Appendix, Formatting\\ \\ \\
+```
+
+
+\pagebreak
+
+## 1. Introduction
```

Comment:
Instead of numbering it manually, you can use `number-sections: true` in the YAML header.